app/views/notifier/mentioned.markerb

```diff
@@ -4,6 +4,12 @@
 <%= t('notifier.mentioned.limited_post') %>
 <% end %>
 
-[<%= t('notifier.comment_on_post.reply', :name => @notification.post_author_name) %>][1]
+<% if @notification.post.is_a?(Post) %>
```

Comment:
I think we should use different templates here. I don't like this if/else here, and now if you are mentioned in a private comment you get a mail with "You were mentioned in a limited post.".